testing/bdd/features/timing/delay.feature

```diff
@@ -68,13 +68,16 @@ Feature: Delay of a document sequence
     | 00:00:05   |              |           |          |          |          | 00:00:10 | 00:00:15           |                  | 00:00:15          |                 | 00:00:15        |               |            |          |          |
```

Comment:
I see. What we discussed is that when we do `document.validate()`, the computed times should be updated, and if they are updated in a correct way, it means that the specified times have been updated correctly by the `RetimingDelayNode`.